<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[test failed!]]>
    </line>
  </reporter-output>
  <suite started-at="2020-05-14T16:30:03Z" name="Data Driven Basics" finished-at="2020-05-14T16:30:13Z" duration-ms="10200">
    <groups>
    </groups>
    <test started-at="2020-05-14T16:30:03Z" name="Bank Manager Login Test" finished-at="2020-05-14T16:30:06Z" duration-ms="3147">
      <class name="com.w2a.testcases.BankManagerLoginTest">
        <test-method signature="loginAsBankManager()[pri:0, instance:com.w2a.testcases.BankManagerLoginTest@5536379e]" started-at="2020-05-14T19:30:03Z" name="loginAsBankManager" finished-at="2020-05-14T19:30:06Z" duration-ms="3128" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[not successful]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: not successful
	at org.testng.Assert.fail(Assert.java:93)
	at com.w2a.testcases.BankManagerLoginTest.loginAsBankManager(BankManagerLoginTest.java:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[test failed!]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginAsBankManager -->
      </class> <!-- com.w2a.testcases.BankManagerLoginTest -->
    </test> <!-- Bank Manager Login Test -->
    <test started-at="2020-05-14T16:30:06Z" name="Add Customer Test" finished-at="2020-05-14T16:30:13Z" duration-ms="7016">
      <class name="com.w2a.testcases.AddCustomerTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.w2a.testcases.AddCustomerTest@24bdb479]" started-at="2020-05-14T19:29:55Z" name="setUp" finished-at="2020-05-14T19:30:03Z" duration-ms="8056" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="addCustomer(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.w2a.testcases.AddCustomerTest@24bdb479]" started-at="2020-05-14T19:30:06Z" name="addCustomer" data-provider="getData" finished-at="2020-05-14T19:30:13Z" duration-ms="6991" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Raman]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Arora]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[A234wd]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Customer added successfully]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addCustomer -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.w2a.testcases.AddCustomerTest@24bdb479]" started-at="2020-05-14T19:30:13Z" name="tearDown" finished-at="2020-05-14T19:30:14Z" duration-ms="1040" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.w2a.testcases.AddCustomerTest -->
    </test> <!-- Add Customer Test -->
  </suite> <!-- Data Driven Basics -->
</testng-results>
